gui/slick/views/schedule.mako

```diff
@@ -233,11 +233,13 @@
             % endif
         % endif
     % elif 'date' == sort:
-        <% cur_ep_airdate = cur_result['localtime'].date() %>
+        <%
+            cur_ep_airdate = cur_result['localtime'].date()
+            cur_ep_enddate = cur_result['localtime'] + datetime.timedelta(minutes = run_time)
+        %>
 
         % if cur_segment != cur_ep_airdate:
             %if run_time:
```

Comment:
@ROpdebee what is the point of this here as you moved line? You should move up too